client/src/ui/components/Spinner.tsx

```diff
@@ -30,6 +30,7 @@ export const Spinner = ({ size, className, ...props }: SpinnerProps) => (
       {
         'border-8 border-t-8 h-24 w-24': size === 'large',
         'border-4 border-t-4 h-8 w-8': size === 'small',
+        'mt-1 border-2 border-t-2 h-4 w-4': size === 'extraSmall',
```

Comment:
mt-1 should probably be removed from here, the other sizes don't have any margin